Makefile

```diff
@@ -46,18 +46,18 @@ download:
 	mkdir -p prototype_source/data
 
 	# transfer learning tutorial data
-	wget -N https://download.pytorch.org/tutorial/hymenoptera_data.zip -P $(DATADIR)
+	wget -q -N https://download.pytorch.org/tutorial/hymenoptera_data.zip -P $(DATADIR)
```

Comment:
Can you please explain why those changes are necessary?

Reply:
This is downloading quietly to avoid the log being too long.

Reply:
Please correct me if I'm wrong, but this change is unrelated to the goal of this PR. So please, do not make in this change, but feel free to file a separate PR/issue if you want to modify this behavior.